src/main/java/seedu/application/logic/parser/Parser.java

```diff
@@ -8,6 +8,9 @@
  */
 public interface Parser<T extends Command> {
 
+    String MESSAGE_UNKNOWN_PREFIX_FOUND = "Unknown prefix detected in the command, please "
```

Comment:
A bit nitpicky but I feel "unexpected" might be a better word to use here
```suggestion
    String MESSAGE_UNEXPECTED_PREFIX_FOUND = "Unexpected prefix detected in the command, please "
```